base/subarray.jl

```diff
@@ -553,13 +553,34 @@ end
 
 Convert every array-slicing operation in the given expression
 (which may be a `begin`/`end` block, loop, function, etc.)
-to return a view.   Scalar indices, non-array types, and
+to return a view. Scalar indices, non-array types, and
 explicit `getindex` calls (as opposed to `array[...]`) are
 unaffected.
 
-Note that the `@views` macro only affects `array[...]` expressions
-that appear explicitly in the given `expression`, not array slicing that
-occurs in functions called by that code.
+!!! note
+    The `@views` macro only affects `array[...]` expressions
+    that appear explicitly in the given `expression`, not array slicing that
+    occurs in functions called by that code.
+
+# Examples
+```jldoctest
+julia> A = zeros(3, 3)
```

Comment:
Perhaps suppress?

Reply:
I wanted to make the change p obvious, I guess. Do you think it's too long?

Reply:
I found the `A = zeros(3, 3)` sufficient, and all other things being equal prefer compactness, hence :).

Reply:
k